src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -134,4 +135,19 @@ public void setSelectedPerson(Person person) {
     public void setSelectedActivity(Activity activity) {
         model.setSelectedActivity(activity);
     }
+
+    @Override
+    public void callAllListFn() {
+        model.resetLists();
+    }
+
+    @Override
+    public boolean modeHasChange_isCurrModeMember() {
+        return model.getAddressBookMode().equals(AppMode.Modes.MEMBER);
```

Comment:
please use proper abstraction getAddressMode.isModeMember(). I think you've written that function. Shouldn't use equals here.

Reply:
Okay will restructure! Thanks Yiqing!